__tests__/app.test.js

```diff
@@ -175,4 +176,56 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/reviews/:review_id/comments", () => {
+    it("200: GET - This test should return one object via the review_id that is being requested in the path.", () => {
+      return request(app)
+        .get("/api/reviews/3/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments[0]).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: expect.any(Number),
+          });
+        });
+    });
+  });
+  describe("GET /api/reviews/:review_id/comments", () => {
+    it("200: GET - This test should return an array of comments based on the given review_id and they should be in order of most recent comments first", () => {
+      return request(app)
+        .get("/api/reviews/3/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          console.log(comments, "I am comments in the test file");
+          expect(comments).toHaveLength(3);
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments).toBeSorted({ descending: true });
+        });
+    });
```

Comment:
you could add a test for 200 : valid ID but no comments and so responds with an empty array